runtime/lua/vim/_init_packages.lua

```diff
@@ -69,6 +69,9 @@ setmetatable(vim, {
         t[key] = val
         return t[key]
       end
+    elseif key == 'version' then
+      t[key] = require('vim.version')
```

Comment:
`version` can now be added to `vim._submodules` instead as it is no longer a special case.